python_bindings/src/PyIROperator.cpp

```diff
@@ -185,6 +185,9 @@ void define_operators(py::module &m) {
     m.def("likely_if_innermost", &likely_if_innermost);
     m.def("saturating_cast", (Expr(*)(Type, Expr)) & saturating_cast);
     m.def("strict_float", &strict_float);
+    m.def("negate", [](const Expr &expr) -> Expr {
```

Comment:
`logical_not` might be a better choice here; `negate` could imply a sign-negation.

Reply:
Thanks! Changed to `logical_not` now. That is a much better name.